frontend/src/components/explorer/interactionPartners/ReactionTable.vue

```diff
@@ -270,4 +279,68 @@ export default {
     }
   }
 }
+table td {
+  text-align: inherit;
+}
+
+@media screen and (max-width: 600px) {
```

Comment:
in `/frontend/src/style/vars.scss` we define mobile as < 660px, why was 600 chosen here?

Reply:
Very good point! We should use ` @media screen and (max-width: $tablet)` here. Implemented in 57ee653